package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "express-rescue",
-  "version": "1.1.32",
+  "version": "1.2.0",
```

Comment:
Please update the package-lock.json as well (running `npm install` again will do that for you`